pilot/test/integration/access.go

```diff
@@ -87,11 +87,12 @@ func (a *accessLogs) check(infra *infra) error {
 				logs := util.FetchLogs(client, pod, ns, container)
 
 				if strings.Contains(logs, "segmentation fault") {
-					return fmt.Errorf("segmentation fault %s", pod)
+					util.CopyCoreFiles(pod, ns, "/etc/istio/proxy/core*", infra.coreFilesDir)
```

Comment:
can you save logs too ?

Reply:
In a separate PR - no need to do everything in a mega PR, we should get this out sooner.

Reply:
the highest priority issue is actually that we aren't getting the actual envoy input, if we did we probably wouldn't need the core dump. we should get both but we should first get the actual output from envoy

Reply:
Not sure if there is any actual output that would show more than that.

Reply:
`strings.Contains(logs,"segmentation fault"` why not print all of logs when there is such segmentation fault